### PR TITLE
ui: Refactor Upstreams and Exposed Paths icons

### PR DESCRIPTION
**Pull request by kaxcode**
- Update Upstreams to have the copy-button to the left of the text
- Update Upstreams to use a definition List
- Update Exposed Paths to use a definition List
Screen Shot 2020-06-17 at 4 54 49 PM
Screen Shot 2020-06-17 at 4 55 01 PM

### CODE REVIEW

File: ui-v2/app/styles/components/composite-row/layout.scss [hunk 1]
```diff
@@ -50,15 +50,12 @@
   display: inline-flex;
   flex-wrap: nowrap;
 }
-%composite-row-detail dt {
```
Comment (kaxcode) [inline ui-v2/app/styles/components/composite-row/layout.scss:53]: Not sure if there was a reasoning for this, but I need to display `dt` to show the icons that also hold the Tooltip.

File: ui-v2/app/styles/components/composite-row/layout.scss [hunk 2]
```diff
@@ -29,7 +29,7 @@
   align-self: center;
 }
 %composite-row-icon {
-  margin-right: 10px;
+  margin-right: 6px;
```
Comment (kaxcode) [inline ui-v2/app/styles/components/composite-row/layout.scss:32]: Discussed with Jasmine to have the spacing between health-check icon and text be 6px, atm. In some frames it's 4px and in other it's 10px.

File: ui-v2/app/styles/components/composite-row/layout.scss (hunk 1 above)
Comment (johncowen) [reply ui-v2/app/styles/components/composite-row/layout.scss:53]: I think in the previous iteration where I'd started adding `dt`s we were actually showing the content of the dt - now that we are this make perfect sense to remove 👍

File: ui-v2/app/components/consul-exposed-path-list/index.hbs [hunk 3]
```diff
@@ -17,52 +17,52 @@
     <div>
       <ul>
       {{#if path.Protocol}}
-        <li class="protocol">
-          <span>
+        <dl class="protocol">
```
Comment (johncowen) [inline ui-v2/app/components/consul-exposed-path-list/index.hbs:20]: I so love this PR! I'm gutted I spotted this - 😿  We should only have `li`'s within a `ul`.

Umm, is it possible to remove the `ul` now we are using `dl`s? The other option would be to put multiple `dt/dd`s inside one single `dl`. That can _sometimes_ cause problems, so I 'generally' opt for multiple `dl`s, but I think maybe we should just use a single `dl` here with multiple `dt/dd`s inside it.

Alternatively, maybe the easiest thing to do here is just remove the `ul`, lets chat about it later, I'm ok with either option.

Comment (kaxcode) [reply ui-v2/app/components/consul-exposed-path-list/index.hbs:20]: I'll try removing the `ul`.

Comment (kaxcode) [reply ui-v2/app/components/consul-exposed-path-list/index.hbs]: Everything works 💯 without the `ul`.

**Pull request merged**